### PR TITLE
docs: add workload collection tutorial

### PR DESCRIPTION
**Pull request by lander2k2**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ You have the following options to install the operator-builder CLI:
 Use wget to download the pre-compiled binaries:
 
 ```bash
-VERSION=v0.6.0
+VERSION=v0.7.0
```
Comment (scottd018) [inline docs/installation.md:15]: I thought @AndriyKalashnykov had fixed these in another branch (version diffs).  Are you needing to rebase, or did I misunderstand what Andriy did?  Just checking to make sure no rebase is needed and that I am indeed correct in that I am misunderstanding.

**Comment by scottd018** [comment]
@lander2k2 just one comment - also don't forget to fixup the commit messages to conform with conventional commits (https://www.conventionalcommits.org/en/v1.0.0/) and force push

File: docs/installation.md (hunk 1 above)
Comment (AndriyKalashnykov) [reply docs/installation.md:15]: @scottd018 i made changes in collection-tutorial branch, no rebase needed. looks i missed that version  ))

Comment (scottd018) [reply docs/installation.md:15]: ah...i see the commits now.  I think I got confused with the brew stuff you fixed last week.

Comment (AndriyKalashnykov) [reply docs/installation.md:15]: @scottd018 no worries, that was different repo

**Pull request merged**